packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -85,7 +85,12 @@ function PickersShortcuts<TValue>(props: PickersShortcutsProps<TValue>) {
         {
           maxHeight: VIEW_HEIGHT,
           maxWidth: 200,
-          overflow: 'auto',
+          scrollbarGutter: 'stable',
+          width: '100%',
```

Comment:
Was this needed? 🤔 
It could impact users with specific shortcut slot configurations. 🙈